packages/cfd/src/Containers/dmt5-trade-modal.tsx

```diff
@@ -61,6 +64,20 @@ const DMT5TradeModal = observer(
             return undefined;
         };
 
+        const mobileURLSet = () => {
+            setURL(window.location.replace(deepLink) as unknown as string);
+            const timeout = setTimeout(() => {
+                if (mobileOSDetect() === 'iOS') {
+                    setURL(window.location.replace(getPlatformMt5DownloadLink('ios')) as unknown as string);
```

Comment:
```suggestion
                    setURL(window.location.replace(getPlatformMt5DownloadLink('ios')));
```
this shouldn't need coercing.... it is returning string no matter what

Reply:
The type of `window.location.replace` is `void`. Making changes to remove coercing

Reply:
why do you need to setURL to a void function @hirad-deriv ? sounds off.

Reply:
Making changes accordingly